server/src/main/java/org/elasticsearch/action/admin/cluster/repositories/get/GetRepositoriesResponse.java

```diff
@@ -49,6 +49,10 @@
         this.repositories = repositories;
     }
 
+    public GetRepositoriesResponse(StreamInput in) throws IOException {
```

Comment:
why add this but not fully convert class (see e.g. AddVotingConfigExclusionsResponse)

Reply:
Not sure I understand, could you please clarify?

Reply:
We generally want to move away from the `Streamable` interface and retire its `readFrom(StreamInput)` method. We have been doing this step-by-step throughout the codebase. The replacement is to define a constructor on the request/response object which takes `StreamInput` as parameter. This also allows us to make many fields in these (Request/Reponse) classes `final` (see e.g. `GetCcrRestoreFileChunkRequest`). The PR here does half of the work in this regard. It adds a constructor that just delegates to the `readFrom` method, i.e. adding more code without the benefit of decluttering the existing one.

Reply:
@ywelsch thanks for the clarification, https://github.com/elastic/elasticsearch/pull/42090/commits/682d583121441f8ee5d1982659d4e1c0b7a4c2ec

Reply:
@ywelsch sorry, I made it wrong first time and fixed here https://github.com/elastic/elasticsearch/pull/42090/commits/e44103294d74d4765d7de5ba91cf13a38417fb50. I have also made similar changes to `GetSnapshotsResponse` as a part of this commit.

Reply:
👍